build.gradle

```diff
@@ -53,10 +54,6 @@ allprojects {
     }
 
     plugins.withId('com.palantir.baseline-error-prone', {
-        dependencies {
-            errorprone 'com.uber.nullaway:nullaway'
-        }
-
```

Comment:
I am going to leave the `NullAway:AnnotatedPackages` & `NullAway:CheckOptionalEmptiness` overrides in place below for now until we can re-enable those in baseline

Reply:
👍